about.md

```diff
@@ -1,5 +1,6 @@
 ---
-title: About
+_title: About
```

Comment:
Since you are adding titles here, you should probably remove them from the toc.yml file so we don't have to keep track of the same string in 2 places.

Reply:
This is sort of a limitation of docfx as far as I'm aware.

The `name:` property in the toc.yml just defines the display name of the link created in the navigation.  There's no way to set the global metadata tags for a document from the toc.

Each non API page that is generated uses what docs calls a **conceptual model** which contains information like the following from the About page:


```json
"rawTitle": "",
"_title": "About",
"title": "History",
"wordCount": 319,
```

Notice that `title` is `History`.
It will set the `title` to the first heading that it finds when parsing the document, whether it's a level 1, 2, or 3 heading.  Doesn't matter, whatever the first heading it.

Then there is the `_title` property which we are setting manually.  This property only affects the title that is displayed in teh tab window, and now as well the title displayed in the OpenGraph and Twitter Card meta data.

As far as I can tell, there is no way to set the title in just the toc or the document and have it affect both the toc and the document.

It's possible to create an extension for the conceptual model processor that could read the `_tocKey` value of the document, then use that to parse the YAML of the toc and extract the `name:` property to set as the `_title` of the document model.

For reference, if you ever want to see the model data generated you can run the command `dotnet docfx build --exportRawModel`, then in the **_site** directory there will be a `*.raw.json` file for each page model